gt-scheduler/website
# Updated new features modal

### Summary

Updated the information modal to discuss the new compare schedules feature instead of recurring events.

### Checklist

### How to Test

Launch the page in a new window where you aren't logged in to see the new modal popup on startup.
